### DISCUSSION TITLE
Implement `node:test` support

### DISCUSSION THREAD

**Discussion by darcyrush** [automock/automock#404] [Ideas]
The native [node:test](https://nodejs.org/api/test.html) module has been available since node 18, with improved test runner support since node 21. I believe that from node 22, it even supports both CJS and ESM mocking. Unlike other libraries, it's mocking module and test runner don't use magic, so testing and developing at the same time is less fatiguing. Moreover, as a native module, it will be supported as long as Node is developed.

Would you please consider adding (at least CJS) support for the native `node:test` module.

**Comment by omermorad** [comment]
Hello @darcyrush,

This will be added soon, I'll let you know :)

**Comment by omermorad** [comment]
@darcyrush would like to create a PR for this?

**Reply by darcyrush** [reply]
I found a very simple alternative solution, so I no longer have the need to use suites

**Reply by omermorad** [reply]
Interesting, can you give an example?
